fluidkeys/main.go

```diff
@@ -31,9 +31,14 @@ func (d DicewarePassword) AsString() string {
 	return strings.Join(d.words, d.separator)
 }
 
+type job struct {
```

Comment:
Could we better define `job`? Is it `generatePgpKey`?

Reply:
Ah no - there's a function called that.... 🤔

Reply:
renamed to `generatePgpKeyResult`